x-pack/plugins/security_solution/public/detections/pages/detection_engine/rules/details/translations.ts

```diff
@@ -28,10 +28,10 @@ export const EXPERIMENTAL = i18n.translate(
   }
 );
 
-export const ACTIVATE_RULE = i18n.translate(
-  'xpack.securitySolution.detectionEngine.ruleDetails.activateRuleLabel',
+export const ACTIVATED_RULE = i18n.translate(
+  'xpack.securitySolution.detectionEngine.ruleDetails.activatedRuleLabel',
   {
-    defaultMessage: 'Activate',
+    defaultMessage: 'Activated',
```

Comment:
Hi @dplumlee, I don't know if this is still possible to do a quick fix. 

I have synced with EUI and the technical writer confirmed the usage of label in `Switch`, there is a [PR](https://github.com/elastic/eui/pull/5119) that updating the EUI guidelines for when to use "past tense" and the behaviour for label ([comment](https://github.com/elastic/eui/pull/5119#discussion_r700386775)):

> In a list of already created items, use the past tense "Enabled". In a single form, use "Enable".

So I think in our case, for Switch to activate a single rule, we probably want to update the label from `Activated` to `Activate` to align with the updated EUI guidelines.
<img width="1792" alt="Rule details page_Switch_Activate" src="https://user-images.githubusercontent.com/81987435/131901142-15adb1a0-c8a2-46ad-a1b4-719423d73913.png">